tgui/packages/tgui/interfaces/PlumbingService.tsx

```diff
@@ -0,0 +1,132 @@
+import { BooleanLike, classes } from 'common/react';
+import { Window } from '../layouts';
+import { useBackend, useLocalState } from '../backend';
+import { Section, Tabs, Button, Stack, Box } from '../components';
+import { ColorItem, LayerSelect } from './RapidPipeDispenser';
+import { capitalizeAll } from 'common/string';
+
+type Data = {
+  categories: Category[];
+  icon: string;
+};
+
+type Category = {
+  cat_name: string;
+  recipes: Recipe[];
+};
+
+type Recipe = {
+  index: number;
+  selected: BooleanLike;
+  name: string;
+};
+
+const PlumbingTypeSection = (props, context) => {
+  const { act, data } = useBackend<Data>(context);
+  const { categories = [] } = data;
+  const [categoryName, setCategoryName] = useLocalState(
+    context,
+    'categoryName',
+    ''
+  );
+  const shownCategory =
+    categories.find((category) => category.cat_name === categoryName) ||
+    categories[0];
+  return (
+    <Section fill scrollable>
+      <Tabs>
+        {categories.map((category, i) => (
```

Comment:
```suggestion
        {categories.map((category) => (
```
no single letter vars. especially if we're not going to use it

Reply:
I actually think using `i` for indexes is good taste, but if this is unused, then it doesn't need to be here.